OpenRA.Mods.Common/Traits/Power/Player/PowerManager.cs

```diff
@@ -54,7 +54,7 @@ public class PowerManager : INotifyCreated, ITick, ISync, IResolveOrder
 
 		public int PowerOutageRemainingTicks { get; private set; }
 		public int PowerOutageTotalTicks { get; private set; }
-		public bool PlayLowPowerNotification { get; set; }
+		public bool PlayLowPowerNotification { get; set; } = true;
```

Comment:
Maybe `AdviceInterval > 0` to make it configurable?

Reply:
makes sense